tf-modules/cumulus-rds-tf/db-provision-lambda.tf

```diff
@@ -0,0 +1,116 @@
+resource "aws_lambda_function" "provision_database" {
```

Comment:
There's duplication here from the ... internal? lambda provisioning work, however decoupling this as a separate module doesn't feel like it makes a ton of sense in terms of usability for end users (this lambda isn't terribly useful as a stand-alone component) and adds a bit of build complexity.....  yet we need a multi-user solution for our CI. 

Open to discussion on how/if to resolve the duplication/inconsistency, up to and including modularization of  the lambda as a separate deployment artifact.

Reply:
More directly *this* file isn't using the *.tf from the /lambda/ source directory, which does some of the same setup/work our CI deployments are.

Reply:
Per our offline convo, let's explore the option of just using the lambda module here directly, given that there doesn't seem to be a functional difference between the resource definitions here vs in the module

Reply:
7d5146053